.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
```

Might wanna add a cache for dependencies to speed up builds
```suggestion
        with:
          cache: 'yarn'
```

---

Is there any way this can be "searched for" and put into a Google sheet for teams to work through? It may also save the Guardian a teensy bit of money.